node/cn/protocol.go

```diff
@@ -74,7 +77,15 @@ const (
 	ReceiptsRequestMsg = 0x0e
 	ReceiptsMsg        = 0x0f
 
-	MsgCodeEnd = 0x10
+	// Protocol messages belonging to klay/64
+	DummyMsg1 = 0x10
+	DummyMsg2 = 0x11 // it is used for consensus message for istanbul engine
```

Comment:
Is 0x10 skipped because there's 0x11? If so, can we make it more explicit?

```suggestion
        Unused10 = 0x10 // Skipped a number because 0x11 is already taken
	Unused11 = 0x11 // Already used by consensus (IstanbulMsg)
```

Reply:
I have updated it as you suggested.